src/components/filopplaster/Filopplaster.tsx

```diff
@@ -123,10 +123,12 @@ const Filopplaster: React.FC<Props> = ({
               withCredentials: true,
               headers: {
                 'Content-Type': 'multipart/form-data',
-                'Content-Transfer-Encoding': 'base64',
                 accept: 'application/json',
                 [HEADER_NAV_CONSUMER_ID]: HEADER_NAV_CONSUMER_ID_VALUE,
               },
+              transformRequest: (data, headers) => {
```

Comment:
Axios har en bug som gjør at formdata ikke blir kjent som formdata. må ha med denne for å hjelpe de litt på vei...